### PR TITLE
Eslint monorepo

### PR DESCRIPTION
**Pull request by Rafal-Doranczyk**
Split Eslint packages into separate projects

### CODE REVIEW

**Review by TheBorsuk — APPROVED** [review]
LGTM 👌

**Review by fabian-grajko — DISMISSED** [review]
👌

File: package.json [hunk 1]
```diff
@@ -17,22 +20,9 @@
   "devDependencies": {
     "eslint": "^7.7.0"
```
Comment (cymruu) [inline package.json:21]: Q: can't we bump eslint version by the occasion?

Comment (Rafal-Doranczyk) [reply package.json]: updated :)

**Review by fabian-grajko — APPROVED** [review]

**Pull request merged**